pinot-segment-local/src/test/resources/data/log4j2.xml

```diff
@@ -29,7 +29,7 @@
   </Appenders>
   <Loggers>
     <AsyncRoot level="warn" additivity="false">
-      <AppenderRef ref="console"/>
+      <AppenderRef ref="console" />
```

Comment:
revert

Reply:
interesting. why this was changed 😂  reverting...